SConstruct

```diff
@@ -2077,23 +2055,6 @@ def postInstallMessage(target, source, env):
               {matlab_ctpath_loc!s}
         """.format(**env_dict))
 
-    if os.name != 'nt':
-        env['setup_cantera'] = pjoin(env['ct_bindir'], 'setup_cantera')
-        env['setup_cantera_csh'] = pjoin(env['ct_bindir'], 'setup_cantera.csh')
-        install_message += textwrap.dedent("""
-
-            Setup scripts to configure the environment for Cantera are at:
-
-              setup script (bash)         {setup_cantera!s}
-              setup script (csh/tcsh)     {setup_cantera_csh!s}
-
-            It is recommended that you run the script for your shell by typing:
-
-              source {setup_cantera!s}
-
-            before using Cantera, or else include its contents in your shell login script.
-        """.format(**env_dict))
-
```

Comment:
It would be nice if there'd be a newline at the end of `install_message`. The current formatting (compiled within conda on fedora) looks like:
```shell
[...]
postInstallMessage(["finish_install"], [])

Cantera has been successfully installed.

File locations:

  library files               /home/ischoegl/.conda/envs/cantera-dev/lib
  C++ headers                 /home/ischoegl/.conda/envs/cantera-dev/include
  samples                     /home/ischoegl/.conda/envs/cantera-dev/share/cantera/samples
  data files                  /home/ischoegl/.conda/envs/cantera-dev/share/cantera/data
  input file converters       /home/ischoegl/.conda/envs/cantera-dev/bin  
  Python package              /home/ischoegl/.conda/envs/cantera-dev/lib/python3.10/site-packages
  Python samples              /home/ischoegl/.conda/envs/cantera-dev/lib/python3.10/site-packages/cantera/examples
scons: done building targets.
```